src/Field/File.php

```diff
@@ -4,14 +4,14 @@
 
 namespace atk4\filestore\Field;
 
-class File extends \atk4\data\Field_SQL
+class File extends \atk4\data\FieldSQL
 {
     use \atk4\core\InitializerTrait {
         init as _init;
     }
 
 
-    public $ui = ['form'=>'\atk4\filestore\FormField\Upload'];
+    public $ui = [['form'=>'\atk4\filestore\Form\Control\Upload']];
```

Comment:
use ::class (but same in reality...) and I think it should be more `['form' => [cl]]` (from seed definition, I do not know this plugin...)

Reply:
Should be `['form' => [\atk4\filestore\Form\Control\Upload::class]]`